packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { first, last } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect, useCallback } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
+
+function KeyboardShorcuts() {
+	const target = useRef( null );
+
+	// Registering the shortcuts
+	const { registerShortcut } = useDispatch( 'core/keyboard-shortcuts' );
+	useEffect( () => {
+		registerShortcut( 'core/block-editor/duplicate', 'block', {
+			modifier: 'primaryShift',
+			character: 'd',
+		} );
+
+		registerShortcut( 'core/block-editor/remove', 'block', {
+			modifier: 'access',
+			character: 'z',
+		} );
+
+		registerShortcut( 'core/block-editor/insertBefore', 'block', {
+			modifier: 'primaryAlt',
+			character: 't',
+		} );
+
+		registerShortcut( 'core/block-editor/insertAfter', 'block', {
+			modifier: 'primaryAlt',
+			character: 'y',
+		} );
+
+		registerShortcut( 'core/block-editor/deleteMultiSelection', 'block', {
+			character: 'del',
+			aliases: [
+				'backspace',
+			],
+		} );
+
+		registerShortcut( 'core/block-editor/selectAll', 'selection', {
+			modifier: 'primary',
+			character: 'a',
+		} );
+
+		registerShortcut( 'core/block-editor/unSelect', 'selections', {
```

Comment:
This one is referred to as `unselect` later in the file.
```suggestion
		registerShortcut( 'core/block-editor/unselect', 'selections', {
```